lib/action_text/attribute.rb

```diff
@@ -27,8 +27,8 @@ def #{name}
             self.rich_text_#{name} ||= ActionText::RichText.new(name: "#{name}", record: self)
           end
 
-          def #{name}=(body)
-            #{name}.body = body
+          def #{name}=(_val)
+            #{name}.body = _val
```

Comment:
What do you think about fully-qualifying the left-hand side of the assignment instead, or even additionally? Otherwise, any argument name we choose is effectively reserved (even though `_val` is less likely than `body` to match the name of a rich text attribute):

```ruby
def #{name}=(body)
  self.#{name}.body = body
end
```

Reply:
done